### ISSUE TITLE
Code reproduction

### ISSUE THREAD

**Issue by 05063112lcs** [maybeLx/MVSFormerPlusPlus#19]
Hello, thank you very much for your excellent work, I am going to reproduce your work in the near future, I see that you are training on 4 A6000GPU with 48GB video memory, I have an A5000 server, can I reproduce it successfully

**Comment by maybeLx** [comment]
Yes, you can decrease the size in the scale_batch_map. Please note that the size in the scale_batch_map should be divisible by the batch size."
